README.md

```diff
@@ -1,4 +1,6 @@
-# RiiConnect24 DNS Server [![Actions Status](https://github.com/RiiConnect24/DNS-Server/workflows/CI/badge.svg)](https://github.com/RiiConnect24/DNS-Server/actions)
+RiiConnect24 DNS Server
+===
+[![Actions Status](https://github.com/RiiConnect24/DNS-Server/workflows/CI/badge.svg)](https://github.com/RiiConnect24/DNS-Server/actions)
```

Comment:
Perhaps revert this? Useless to have an empty line only for the badge

Reply:
Additionally please change `/CI/badge.svg` to `/Build/badge.svg`, this causes a conflict as a result of recent commit.